extensions/amp-animation/0.1/test-e2e/test-amp-animation.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'amp-animation',
+  {
+    testUrl:
+      'http://localhost:8000/test/fixtures/e2e/amp-animation/simple.html',
+    environments: 'ampdoc-amp4ads-preset',
+  },
+  env => {
+    let controller;
+
+    beforeEach(async () => {
+      controller = env.controller;
+    });
+
+    it('simple animation', async () => {
```

Comment:
Is it enough coverage to test a simple animation alone or should more complex tests be added as well?

Reply:
Yeah, I think we will.